docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Integral numeric types - C# reference"
+description: "Learn the range, storage size, and uses for each of the integral numeric types."
+ms.date: 06/25/2019
+f1_keywords:
+  - "byte"
+  - "byte_CSharpKeyword"
+  - "sbyte_CSharpKeyword"
+  - "sbyte"
+  - "short"
+  - "short_CSharpKeyword"
+  - "ushort"
+  - "ushort_CSharpKeyword"
+  - "int_CSharpKeyword"
+  - "int"
+  - "uint"
+  - "uint_CSharpKeyword"
+  - "long_CSharpKeyword"
+  - "long"
+  - "ulong_CSharpKeyword"
+  - "ulong"
+helpviewer_keywords: 
+  - "integral types, C#"
+  - "Visual C#, integral types"
+  - "types [C#], integral types"
+  - "ranges of integral types [C#]"
+  - "byte keyword [C#]"
+  - "sbyte keyword [C#]"
+  - "short keyword [C#]"
+  - "ushort keyword [C#]"
+  - "int keyword [C#]"
+  - "uint keyword [C#]"
+  - "long keyword [C#]"
+  - "ulong keyword [C#]"
+---
+# Integral numeric types  (C# reference)
+
+The **integral types** are a subset of the **numeric types**. The **numeric types** are a subset of the **simple types**. All integral types are value types and can be initialized with *literals*, for example "24". All integral types support [arithmetic](../operators/arithmetic-operators.md), [bitwise logical](../operators/bitwise-and-shift-operators.md), [comparison, and [equality](../operators/equality-operators.md) operators.
```

Comment:
I was wrong in my first edit to mention value types and literals in one sentence. Literals should come with simple types. Also removed the "[" typo before the "equality" word.

```suggestion
The **integral numeric types** are a subset of the **simple types** and can be initialized with [*literals*](#integral-literals). All integral types are also value types.

All integral numeric types support [arithmetic](../operators/arithmetic-operators.md), [bitwise logical](../operators/bitwise-and-shift-operators.md), [comparison, and equality](../operators/equality-operators.md) operators.
```